src/viewer.ts

```diff
@@ -11,6 +11,7 @@ interface Position {}
 
 interface Client {
     type: 'viewer' | 'tab'
+    prevType?: 'viewer' | 'tab'
```

Comment:
This file change is by accident. The main purpose is to retain preview type when user accidentally refresh pdf viewer. Not quite related with the PR.